src/Events/Span.php

```diff
@@ -0,0 +1,40 @@
+<?php
+namespace AG\ElasticApmLaravel\Events;
+
+use PhilKra\Events\Span as BaseSpan;
+
+class Span extends BaseSpan
```

Comment:
I created a [PR](https://github.com/philkra/elastic-apm-php-agent/pull/121) to avoid this situation. Until then, there is no other way of setting the start time of a Span.